hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketCache.java

```diff
@@ -242,6 +244,11 @@ public int compare(BlockCacheKey a, BlockCacheKey b) {
   /** In-memory bucket size */
   private float memoryFactor;
 
+  private String[] filePaths;
+  static final String FILE_VERIFY_ALGORITHM = "hbase.bucketcache.file.verify.algorithm";
```

Comment:
Please rename this parameters, "hbase.bucketcache.persistent.file.integrity.check.algorithm" for your reference.